src/servicehandler.h

```diff
@@ -76,75 +74,60 @@ class MKCAL_EXPORT ServiceHandler : QObject
     /** Send the invitation to the list of people stated as attendees.
       It would load the appropriate plugin to do it, and if there
```

Comment:
Bonus points for removing these repeating "it would load..." lines which are not even good English :)

Reply:
Sure, I've removed these lines and added a brief comment in the ServiceHandler class definition mentioning that plugins are used based on the `Notebook::pluginName()` return.